packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -546,6 +692,8 @@ export const DialogueMatchingPage = ({classes}: {
     collectionName: 'Users'  
   });
 
+  if (!usersOptedInToDialogueFacilitation) return <p>Error...</p>
```

Comment:
This will probably be true while it's loading!  I'd only return this if `!loading` (and return `<Loading />` otherwise)